src/GovUk.Education.ExploreEducationStatistics.Content.Model/Predicates/ReleaseVersionPredicates.cs

```diff
@@ -47,4 +47,26 @@ public static class ReleaseVersionPredicates
                 maxVersion => maxVersion,
                 (releaseVersion, _) => releaseVersion);
     }
+
+    /// <summary>
+    /// Filters a sequence of <see cref="IQueryable{T}"/> of type <see cref="ReleaseVersion"/> to only include the latest
+    /// published versions of each release.
+    /// </summary>
+    /// <param name="releaseVersionsQueryable">The source <see cref="IQueryable{T}"/> of type <see cref="ReleaseVersion"/> to filter.</param>
+    /// <param name="releaseId">Unique identifier of a release to filter by.</param>
+    /// <param name="publishedOnly">Flag to only include published release versions.</param>
+    /// <returns>An <see cref="IQueryable{T}"/> of type <see cref="ReleaseVersion"/> that contains elements from the input
+    /// sequence filtered to only include the latest versions of each release.</returns>
```

Comment:
The `summary` and `returns` descriptions aren't consistent with the method which can only return at most one release version.